ui/client/components/graph/node-modal/NodeAdditionalInfoBox.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from 'react';
+import HttpService from "../../../http/HttpService";
+import ReactMarkdown from 'react-markdown/with-html';
+import "../../../stylesheets/markdown.styl"
+
+type Props = {
+  processId: string,
+  node: any
+}
+
+//Types should match implementations of NodeAdditionalInfo on Backend!
+type NodeAdditionalInfo = MarkdownNodeAdditionalInfo
+
+type MarkdownNodeAdditionalInfo = {
+    type: "MarkdownNodeAdditionalInfo",
+    content: string
+}
+
+
+export default function NodeAdditionalInfoBox(props: Props) {
+
+    const [additionalData, setAdditionalData] = useState<NodeAdditionalInfo>(null)
+    const {
+      processId, node
+    } = props
+
+    useEffect(() => {
+        if (node?.type) {
+            HttpService.getNodeAdditionalData(processId, node).then(res => setAdditionalData(res.data))
```

Comment:
I think it's not good idea.. You should speak with @JulianWielga  about this. There should be action call.

Reply:
I've talked with him, we agreed that if it's local state for this component there's no real need for redux. I added debounce, since we don't want to fetch too early